qt/aqt/reviewer.py

```diff
@@ -364,6 +388,28 @@ def _showQuestion(self) -> None:
         # user hook
         gui_hooks.reviewer_did_show_question(c)
 
+        conf = self.mw.col.decks.config_dict_for_deck_id(self.card.current_deck_id())
+        timer = None
+
+        if conf["secondsToShowAnswer"]:
+            timer = self._show_answer_timer = self.mw.progress.timer(
```

Comment:
Any particular reason you didn't use single_shot() here?

Reply:
This is to compare the timers so that we make sure a previous card's timer doesn't get to execute on the next card.